spec/controllers/state_file/questions/nj_retirement_warning_controller_spec.rb

```diff
@@ -83,4 +83,28 @@
       end
     end
   end
+
+  describe "eligibility_offboarding_concern" do
+    # the disqualifying param here is permanent_address_outside_md but this is set in the form based on:
+    # * whether they are using the address from DF
+    # * if so, whether that address is in MD
+    # so we have to mock the DF data
+    it_behaves_like :eligibility_offboarding_concern, intake_factory: :state_file_nj_intake do
```

Comment:
i don’t think this comment applies here because it’s nj and not md? but i may be wrong.

Reply:
oops! i thought i deleted the comment from all places but obviously forgot one.